main.py

```diff
@@ -2,108 +2,112 @@
 
 def highestVaccines():
 
-    with session.transaction(TransactionType.READ) as read_transaction:
+    read_transaction = session.transaction(TransactionType.READ)
 
-        answer_iterator = read_transaction.query().match_group_aggregate('match $p isa person, has certificate_no $no, has country $c; get $p, $no, $c; group $c; count;')
+    answer_iterator = read_transaction.query().match_group_aggregate('match $p isa person, has certificate_no $no, has country $c; get $p, $no, $c; group $c; count;')
 
-        stats = []
+    stats = []
 
-        for answer in answer_iterator:
-            stats.append([answer._numeric._int_value, answer._owner._value])
+    for answer in answer_iterator:
+        stats.append([answer._numeric._int_value, answer._owner._value])
 
-        stats.sort(reverse = True)
+    stats.sort(reverse = True)
 
-        print("\nTop 3 countries with most Vaccinations:")
-        for i in range(3):
-            print(stats[i][1], ": ", stats[i][0])
+    if len(stats) < 3:
+        print("\nEntries for only", len(stats), "countries present!")
+        print("Top", len(stats), "countries with most Vaccinations:")
+        for country_stat in stats:
+            print(country_stat[1], ":", country_stat[0])
+        return

```

Comment:
I don't think this output will look good in the case where `len(stats) == 0` - it's worth having a case for that.